### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by msricher**
Updates the README to be consistent with other QC-Devs projects.

I moved the installation notes into the website and linked to it from the README.

Since there is no release yet, and CodeFactor is not integrated, I have commented out those "badge" images in the README header for now.

## Checklist

- [x] Write a good description of what the PR does.
- [x] Add tests for each unit of code added (e.g. function, class)
- [x] Update documentation
- [x] Squash commits that can be grouped together
- [ ] Rebase onto master

## Type of Changes

|   | Type |
| ------------- | ------------- |
| ✓  | :bug: Bug fix  |
| ✓  | :sparkles: New feature |
| ✓  | :hammer: Refactoring  |
| ✓  | :scroll: Docs |

### CODE REVIEW

**Review by PaulWAyers — COMMENTED** [review]
I would say that this is a partial list of features, as the Readme may rarely be entirely up-to-date. Perhaps say, instead,
## Selected Features

or 

## Features (Partial List)

**Comment by msricher** [comment]
> I would say that this is a partial list of features, as the Readme may rarely be entirely up-to-date. Perhaps say, instead,
> ## Selected Features
> 
> or
> ## Features (Partial List)

Done. I also linked some parts to the website.

File: README.md [hunk 1]
```diff
@@ -1,95 +1,61 @@
...
-```
-and to test the interface to `iodata`, run
-```bash
-tox -e iodata
-```
-
-## Features
-Following features are supported in `gbasis`:
+
+[See the website for installation instructions.](https://gbasis.qcdevs.org/installation.html)
```
Comment (FarnazH) [inline README.md:36]: @msricher, before referring to the website, please include instructions for `pip` installation similar to `grid` and `iodata`? In the same fashion, make sure to use `qc-gbasis`.

**Review by FarnazH — COMMENTED** [review]
@msricher thanks for your PR. I made some updates to the README. I wonder whether we should keep a list of features in the README. The webpage can include a concise feature list on its 1st page, but similar to `iodata` and `grid`, it would be good to delegate such information to the website. What do you think @PaulWAyers  and @leila-pujal?

File: README.md (hunk 1 above)
Comment (PaulWAyers) [reply README.md:36]: I'm OK with a full list in the readme, but it's essential for the web site. `libcint` does a full(ish) list in the readme, so there is precedent for that. I think the readme has a different (more technical) audience than the web site.

My hesitation at making a full list in the readme is that then we have to update not just the web site but also the readme as we update the package. It's simpler to just update one of them, and in that case the readme should list selected features, not everything, and make it clear that it is just the selected web sites.

**Comment by FarnazH** [comment]
I am merging this because the current README is outdated and wrong in some instances.

**Review by FarnazH — APPROVED** [review]
I am merging this because the current README is outdated and wrong in some instances.

**Pull request merged**